### PR TITLE
Added Isomorphic Material Relay Starter Kit

### PR DESCRIPTION
**Pull request by thelordoftheboards**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ Awesome resources for [Relay](https://github.com/facebook/relay), based on the [
 - [Relay Skeleton](https://github.com/fortruce/relay-skeleton) - React, Relay, GraphQL project skeleton.
 - [Relay Starter Kit](https://github.com/relayjs/relay-starter-kit) - An app that it already set up with a basic setup. Just clone and tweak to suit your needs!
 - [Simple Relay Starter](https://github.com/mhart/simple-relay-starter) - A Browserify version of the [Relay Starter Kit](https://github.com/relayjs/relay-starter-kit).
+- [Isomorphic Material Relay Starter Kit](https://github.com/codefoundries/isomorphic-material-relay-starter-kit) - Isomorphic React, Relay, GraphQL starter kit with examples. Uses Material-UI, Webpack, hot reload, Helmet, JWT. Utilizes Facebook Data Loader for persistence into in-memory structures and Cassandra.
```
Comment (expede) [inline README.md:91]: Perhaps a bit pedantic, but Relay assumes React and GraphQL, so there's no need to mention them in the description :wink:

**Comment by expede** [comment]
Just the one comment, but this is otherwise good to go! :smile:

**Comment by thelordoftheboards** [comment]
This makes perfect sense to me. The people who are looking most probably already know what the relation between React, GraphQL and Relay is. Should I close this pull request and submit a different one, or you can change it?

**Comment by expede** [comment]
@thelordoftheboards To save effort, I'll go ahead and merge this and then adjust it :smile:

**Comment by thelordoftheboards** [comment]
@expede Thank you.

**Pull request merged**